test/components/views/messages/MPollBody-test.tsx

```diff
@@ -456,13 +470,13 @@ describe("MPollBody", () => {
             responseEvent("@catrd:example.com", "poutine"),
             responseEvent("@dune2:example.com", "wings"),
         ];
-        const body = newMPollBody(votes, [], null, false);
+        const { container } = newMPollBody(votes, [], undefined, false);
 
         // My vote is marked
-        expect(body.find('input[value="pizza"]').prop("checked")).toBeTruthy();
+        expect(container.querySelector('input[value="pizza"]')!.hasAttribute("checked")).toBeTruthy();
```

Comment:
Recently discovered that we can now do

```suggestion
        expect(container.querySelector('input[value="pizza"]')!).toHaveAttribute("checked", "true");
```

or

```suggestion
        expect(container.querySelector('input[value="pizza"]')!).toBeChecked();
```

Reply:
TIL! thanks